src/Server.hs

```diff
@@ -0,0 +1,29 @@
+{-# LANGUAGE DataKinds     #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE RankNTypes    #-}
+{-# LANGUAGE TypeOperators #-}
+
+module Server (api, app) where
+
+import Data.Proxy                 (Proxy (..))
+
+import Servant
+
+import Servant.API.Generic
+import Servant.Server.Generic
+
+data Routes route = Routes {
```

Comment:
I guess it should be `newtype`.
It doesn't matter at the moment, but `hlint` would warn about it

Reply:
Also better to name this data type `HintmanSite`. It's actually nice to have a naming convention for `servant` stuff. I will describe it in the next comments as well. The first rule: route data types should have `Site` suffix.